common/bitcointree/musig2_test.go

```diff
@@ -26,110 +27,107 @@ var lifetime = common.RelativeLocktime{Type: common.LocktimeTypeBlock, Value: 14
 var testTxid, _ = chainhash.NewHashFromStr("49f8664acc899be91902f8ade781b7eeb9cbe22bdd9efbc36e56195de21bcd12")
 
 func TestRoundTripSignTree(t *testing.T) {
+	t.Parallel()
 	fixtures := parseFixtures(t)
-	for _, f := range fixtures.Valid {
-		// Generate 20 cosigners
-		cosigners := make([]*secp256k1.PrivateKey, 20)
-		cosignerPubkeys := make([]*btcec.PublicKey, 20)
-		for i := 0; i < 20; i++ {
-			prvkey, err := secp256k1.GeneratePrivateKey()
+	for i, f := range fixtures.Valid {
+		t.Run(fmt.Sprintf("test %d", i), func(t *testing.T) {
```

Comment:
let's give each test vector a name instead of calling them `test_0`, `test_1`, ...